AK/Utf16View.cpp

```diff
@@ -300,6 +300,22 @@ bool Utf16View::starts_with(Utf16View const& needle) const
     return true;
 }
 
+// https://infra.spec.whatwg.org/#code-unit-less-than
+bool Utf16View::is_code_unit_less_than(Utf16View const& other) const
+{
+    auto a = m_code_units;
+    auto b = other.m_code_units;
+
+    auto common_length = min(a.size(), b.size());
+
+    for (size_t position = 0; position < common_length; ++position) {
+        if (a[position] != b[position])
+            return a[position] < b[position];
+    }
+
+    return a.size() < b.size();
+}
```

Comment:
Missing spec steps?

Reply:
I did think about it, but it doesn't follow the infra steps very closely, so I figured I would just put the spec link instead. Happy to just paste in a block of comments though if better

Reply:
Having them there is better than not imo. 🤷

Reply:
Spec steps feel like they would just be noise here